### ISSUE TITLE
Kuldge data/nodes: fake node for begin and end of course and conventionalized node layout positions

### ISSUE THREAD

**Issue by mbaljko** [LePa-YU/Visualizer#13]
-datasets should have "begin course" ER (e.g., course description or syllabus) and "end course" (e.g., "farewell email" or some other thing that gets generated at the end of the course"  
- these two special ERs gets picked up by the visualizer and get represented by two special nodes, Begin Node (BN) and End Node (EN)
- the BN and EN nodes get programmatically assigned positions in the layout - 
- assume (0,0) is lower left course, BN is in position (0, max/2) (left edge, midway between top and bottom)
- EN is in position (max, max/2) (right edge, midway between top and bottom)
- then the remainder of the nodes get automatically laid out with physics between BN and EN
- this will kinda-sorta create a "tablature" layout, kinda-sorta approximating a gantt chart

**Comment by ElmiraOn** [comment]
Hi Melanie, 

I checked and it is possible to have BN and EN nodes set at specific positions. I set the BN at x = -1000, y = 0 and EN at x = 1000, y = 0. The result looks like this:
![image](https://user-images.githubusercontent.com/90576588/208838888-069ff719-52c9-4847-9793-b1757566fbdb.png)


The x values are roughly based on the width of the area for the network and my assumption is that both nodes are part of the dataset (currently they are derived from the CSV input) with type "start" and "end" respectively (Their displayed title can be anything). at the moment I am using a "diamond" to represent the nodes in the same color as iERs
